src/main/java/org/folio/circulation/domain/TimePeriod.java

```diff
@@ -0,0 +1,38 @@
+package org.folio.circulation.domain;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getIntegerProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import java.time.temporal.ChronoUnit;
+
+import io.vertx.core.json.JsonObject;
+
+public class TimePeriod {
```

Comment:
This appears to overlap significantly with the [period class](https://github.com/folio-org/mod-circulation/blob/master/src/main/java/org/folio/circulation/domain/policy/Period.java). 

It may be that these could be consolidated in the future (but likely not today)?

Reply:
D'oh! I didn't notice that class in the policy directory. My bad. I agree, let's do the consolidation in the future.